london.md

```diff
@@ -0,0 +1,102 @@
+# London
+
+London is a large and diverse city, made up of many different neighborhoods,
+each with something unique to offer.
+
+Most of the technology scene is centered around East-London, with a lot of
+design-only firms around Oxford Circus.
+
+## Coffee shops
+
+These places have good coffee, good WiFi, and are easily accessible.
+
+* Long Lane: [Dose](http://dose-coffee.com)
+
+Slightly further from the office:
+
+* Clerkenwell Road: [Workshop](http://www.workshopcoffee.com)
+* Clerkenwell Road: [J&A](http://jandacafe.com/)
+* Old Street: [Ozone Coffee](http://www.ozonecoffee.co.uk)
+* Old Street: [Timberyard](https://tyuk.com/locations/ty-old-street/)
+
+## Food
+
+* Long Lane: [Kurumaya](http://kurumaya.co.uk) (Japanese)
+* Saint John Street: [Benito's Hat](http://benitos-hat.com) (Mexican)
+* Saint John Street: [Pixxa](http://www.mypixxa.com) (Italian)
+* Saint John Street: [Pho](http://www.phocafe.co.uk) (Vietnamese)
+* Goswell Road: Pasta Cosi (Italian)
+* Cowcross Street: [Bar Burrito](http://barburrito.co.uk) (Mexican)
+* Cowcross Street: [Leon](http://leonrestaurants.co.uk) (Modern European)
+* Charterhouse Street: [Best Mangal](http://www.thebestmangal.com) (Middle Eastern)
+* Smithfield Street: [Caphe Pho](http://www.caphepho.co.uk) (Vietnamese)
+* Whitecross Street: [Pham Sushi](http://www.phamsushi.co.uk) (Japanese)
+
+There are also several good food markets near the office, on:
+
+* Leather Lane
+* Whitecross Street
+
+## Pubs
+
+* Long Lane: [Old Red Cow](http://theoldredcow.com)
+* Charterhouse Street: [Fox and Anchor](http://www.foxandanchor.com)
+* Middle Street: Hand and Sheers
+
+## Event spaces 
+
+## Coworking spaces
+
+* Old Street/Shoreditch: [Central Working](http://centralworking.com) (our
+  co-working space)
+* Old Street/Shoreditch: [Google Campus](https://www.campus.co/london/en), a
+  free space for the community, provided by Google.
+* St. Katharine's Docks:
+  [Rainmaking Loft](http://www.rainmakingloft.com/london/welcome)
+* Whitechapel: [Central Working](http://centralworking.com), big, and has a
+  great event space for members.
+
+## Hotels
+
+If you're looking last-minute, use [Hotel Tonight](http://www.hoteltonight.com).
+
+## User groups
+
+We've been to and can recommend:
+
+* [London Ruby Users Group (LRUG)](http://lrug.org)
+
+Less tech, more meet-and-greet:
+
+* [Pub Standards](https://london.pubstandards.com), every 3rd Thursday of the
+  month Wednesday morning at [The Bricklayers Arms](http://maps.google.co.uk/maps?q=31%20Gresse%20Street,%20London%20W1T%201QS)
+* [Thirst Fridays](https://generalassemb.ly/education/thirst-fridays/london),
+  the first Friday of every month.
+
+## Designers and developers
+
+Some folks to follow:
+
+* [London Dribbblers](http://dribbble.com/designers?location=London)
+* [London Ruby
+  GitHubbers](https://github.com/search?type=Users&language=ruby&q=location:london)
+* [Boston Swift
```

Comment:
Sorry Dan, I merged already (probably a bit hastily judging by feedback).

Can you open a new PR?

On Wed, Dec 2, 2015 at 11:21 AM Dan Barber <notifications@github.com> wrote:

> In london.md
> <https://github.com/thoughtbot/cities/pull/12#discussion_r46403064>:
>
> > +
> > +Less tech, more meet-and-greet:
> > +
> > +* [Pub Standards](https://london.pubstandards.com), every 3rd Thursday of the
> > +  month Wednesday morning at [The Bricklayers Arms](http://maps.google.co.uk/maps?q=31%20Gresse%20Street,%20London%20W1T%201QS)
> > +* [Thirst Fridays](https://generalassemb.ly/education/thirst-fridays/london),
> > +  the first Friday of every month.
> > +
> > +## Designers and developers
> > +
> > +Some folks to follow:
> > +
> > +* [London Dribbblers](http://dribbble.com/designers?location=London)
> > +* [London Ruby
> > +  GitHubbers](https://github.com/search?type=Users&language=ruby&q=location:london)
> > +* [Boston Swift
>
> Should this say "London Swift"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/cities/pull/12/files#r46403064>.
>